src/lib.rs

```diff
@@ -143,33 +191,21 @@ struct DebugInfo<'object, Endian>
 {
     debug_line: gimli::DebugLine<'object, Endian>,
     units: Vec<Unit<'object, Endian>>,
+    with_demangling: bool,
```

Comment:
This should probably carry on the full `Options`; we may want to add things to it later.